src/main/java/net/syscon/elite/api/model/PrisonerDetail.java

```diff
@@ -24,73 +21,108 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @EqualsAndHashCode
+@Data
 public class PrisonerDetail {
     @JsonIgnore
     private Map<String, Object> additionalProperties;
 
+    @ApiModelProperty(required = true, value = "The prisoner's unique offender number (aka NOMS Number in the UK).")
     @NotBlank
     private String offenderNo;
 
+    @ApiModelProperty(value = "The prisoner's title.")
     private String title;
 
+    @ApiModelProperty(value = "The prisoner's name suffix.")
     private String suffix;
 
+    @ApiModelProperty(required = true, value = "The prisoner's first name.")
     @NotBlank
     private String firstName;
 
+    @ApiModelProperty(value = "The prisoner's middle name(s).")
     private String middleNames;
 
+    @ApiModelProperty(required = true, value = "The prisoner's last name.")
     @NotBlank
     private String lastName;
 
+    @ApiModelProperty(required = true, value = "The prisoner's date of birth (in YYYY-MM-DD format).")
     @NotNull
     private LocalDate dateOfBirth;
 
+    @ApiModelProperty(required = true, value = "The prisoner's gender.")
     @NotBlank
     private String gender;
 
+    @ApiModelProperty(required = true, value = "The prisoner's gender code.")
     @NotBlank
     private String sexCode;
```

Comment:
so they do it for this anyway which is good!

Reply:
Nice to give an example